### PR TITLE
implement onBackPressed() in MainActivity ( sample Code )

### PR DESCRIPTION
**Pull request by rezadarvishian**

### CODE REVIEW

**Comment by Canato** [comment]
@rezadarvishian can you please check the broke CI?

File: sample/src/main/java/com/canhub/cropper/sample/SMainActivity.kt [hunk 1]
```diff
@@ -31,24 +29,27 @@ internal class SMainActivity : AppCompatActivity() {
...
                 .replace(R.id.container, SCropImageFragmentJava.newInstance())
                 .commit()
         }
     }
 
-    private fun hideButtons(binding: ActivityMainBinding) {
-        binding.image.visibility = View.GONE
-        binding.buttons.visibility = View.GONE
+    override fun onBackPressed() {
+        if (supportFragmentManager.findFragmentById(R.id.container) != null) {
```
Comment (Canato) [inline sample/src/main/java/com/canhub/cropper/sample/SMainActivity.kt:47]: Let's use the view binding, like `binding.container`. We will need to make `binding` value private in the class, like `private var binding: ActivityMainBinding? = null`

**Review by Canato — CHANGES_REQUESTED** [review]
When we click the options `Sample calling cropimage` and `sample calling cropimage java` we have a white background, please check this.

File: sample/src/main/java/com/canhub/cropper/sample/SMainActivity.kt (hunk 1 above)
Comment (rezadarvishian) [reply sample/src/main/java/com/canhub/cropper/sample/SMainActivity.kt:47]: What do you mean "we have a white background"? Is there a problem displaying the fragment or is it wrong to use a white background?

**Comment by Canato** [comment]
> What do you mean "we have a white background"? Is there a problem displaying the fragment or is it wrong to use a white background?

Previous behaviour: 
Screenshot_20210812-082033

This PR behaviour:
Screenshot_20210812-081602

There is no need to change this, we should keep consistent

**Review by Canato — CHANGES_REQUESTED** [review]
Heey thanks for the binding changes!! 

but =(, CI is broke because of Ktlint, please check this when you have time =D

**Comment by rezadarvishian** [comment]
> 
> 
> > What do you mean "we have a white background"? Is there a problem displaying the fragment or is it wrong to use a white background?
> 
> Previous behaviour:
> 
> 
> This PR behaviour:
> 
> 
> There is no need to change this, we should keep consistent

this happen because your device dark mode is enabled! so i changed xml background color to colorOnPrimary for set background color depends on dark mode state.

**Comment by rezadarvishian** [comment]
> 
> 
> Heey thanks for the binding changes!!
> 
> but =(, CI is broke because of Ktlint, please check this when you have time =D

Excuse me, but can you explain to me why it has to fail because of an empty space? Is this so important?

**Review by Canato — APPROVED** [review]

**Comment by rezadarvishian** [comment]
> **And is not like this is a huge time changer for the developer.** As describe on out [CONTRIBUTING guide](https://github.com/CanHub/Android-Image-Cropper/blob/main/CONTRIBUTING.md) the developer just need to run `./gradlew ktlintFormat` before the final commit. One simple command that improve a lot our code base and it consistency.


Oh, I'm sorry, it was my fault that I didn't read the complete [CONTRIBUTING guide]. Thanks for the explanation. good luck

**Pull request merged**